### PR TITLE
[Inductor] Enable accuracy test for CPPBackend

### PR DESCRIPTION
**Pull request by EikanWang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #94898

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @Guobing-Chen @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

**Comment by EikanWang** [comment]
@desertfire , may I know if I have captured your point to enable the accuracy test for CppBackend?

**Comment by desertfire** [comment]
> @desertfire , may I know if I have captured your point to enable the accuracy test for CppBackend?

Yes, you are on the right track. It may take a few iterations to get all the configs right. You will also need to create a separate CI skip list for CPU, https://github.com/pytorch/pytorch/blob/3ace14eb8b5e437322acf962d2f170561fd4e3bc/benchmarks/dynamo/common.py#L69

**Review by jgong5 — APPROVED** [review]

File: .ci/pytorch/test.sh [hunk 1]
```diff
@@ -374,6 +379,10 @@ test_inductor_huggingface() {
   test_inductor_benchmark huggingface ""
 }
 
+test_inductor_huggingface_cpu_accuracy() {
```
Comment (desertfire) [inline .ci/pytorch/test.sh:382]: Can we combine this with `test_inductor_huggingface` and add a device parameter? The same for the other two benchmarks.

**Review by desertfire — APPROVED** [review]
Please address my comments before landing.

**Comment by ezyang** [comment]
Can we please stop adding O(number of benchmark suite) changes to the test.sh file. It makes it a lot more difficult to make changes to the test script, as you keep increasing the size of the script by a factor of three for every change you make. In this case, cpu/cuda could have been handled generically, either inside `test_inductor_benchmark` or as flags passed to the benchmark suite. I'll submit a refactor PR for this, but please tag me on these PRs on in the future.

**Pull request closed without merging**